app/src/main/java/com/eveningoutpost/dexdrip/NavDrawerBuilder.java

```diff
@@ -102,15 +108,22 @@ public NavDrawerBuilder(final Context context) {
                 this.nav_drawer_options.add(context.getString(R.string.start_sensor));
                 this.nav_drawer_intents.add(new Intent(context, StartNewSensor.class));
             }
-            else if (!Pref.getBooleanDefaultFalse("engineering_mode") && is_active_sensor) {
-                this.nav_drawer_options.add(context.getString(R.string.stop_sensor));
-                this.nav_drawer_intents.add(new Intent(context, StopSensor.class));
-            }
-            else {
-                this.nav_drawer_options.add(context.getString(R.string.start_sensor));
-                this.nav_drawer_intents.add(new Intent(context, StartNewSensor.class));
+
+            else if (!Pref.getBooleanDefaultFalse("engineering_mode")) {
+                if (!(DexCollectionType.getDexCollectionType().equals(Follower) || DexCollectionType.getDexCollectionType().equals(NSFollow) || DexCollectionType.getDexCollectionType().equals(SHFollow)
```

Comment:
I recommend to add an additional HashSet to `DexCollectionType` containing devices which support sensor starts/stops as well as a function `hasSensorStartStopSupport`. This function can than called instead of the logical ORs.